go/vt/servenv/status.go

```diff
@@ -75,6 +75,9 @@ var (
 	blockProfileRate = 0
 
 	globalStatus = newStatusPage("")
+	// tableRefreshRate defines the time interval in milliseconds of tables get updated,
+	// and only update tables those have "refreshRequired" class
+	tableRefreshRate = 10000
```

Comment:
Regarding the comment I wrote on https://github.com/vitessio/vitess/pull/12316#discussion_r1104055991, I am still unsure if we should make this value configurable through a CLI flag. @GuptaManan100, wdyt?

Reply:
I think it makes sense to have this as a flag. The default being a very large value so that the current behaviour remains unchanged and is opt-in.

Reply:
Thanks @GuptaManan100 and @frouioui, so I guess this is the right place to go? https://github.com/vitessio/vitess/blob/main/go/vt/servenv/servenv.go#L92

Also, I would love to hear your opinions on what you think a default value is. TBH I think 10 seconds is relatively a conservative number, I probably would set it to 5 in our vitess clusters.

Reply:
I think both ways are fine, we let the user configure it anyway if they're not happy with the default.

Reply:
`servenv.go` looks like a good place for me to add the flag 👍🏻

Reply:
However, watch out, the `TestHelpOutput` test in `go/flags/endtoend` will fail since you are adding a flag to several binaries. What the test does is to execute `<all vitess binaries> --help` and compare the output with the `.txt` files stored at the root of the same package. You will need to update the `.txt`s files with the new list of flags.

Reply:
However, watch out, the `TestHelpOutput` test in `go/flags/endtoend` will fail since you are adding a flag to several binaries. What the test does is to execute `<all vitess binaries> --help` and compare the output with the `.txt` files stored at the root of the same package. You will need to update the `.txt`s files with the new list of flags.

Reply:
I think the default should be very high like `10000000` seconds, so that the behaviour is disabled by default. If someone wants the refresh, then they can do it be adding the flag. WDYT @frouioui?

Reply:
@GuptaManan100 i like the idea of having it disabled by default but in that case let's use a value such as `0` that means "no refresh"